webpack.config.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
https://www.npmjs.com/package/generator-openmrs-owa . We used OpenMRS Open Web App Generator